packages/docusaurus/bin/docusaurus.js

```diff
@@ -94,6 +95,28 @@ program
     });
   });
 
+program
+  .command('info')
+  .description('Shows debugging information about the local environment')
+  .action(() => {
+    console.log(chalk.bold('\nEnvironment Info:'));
+    envinfo
+      .run(
+        {
+          System: ['OS', 'CPU'],
+          Binaries: ['Node', 'Yarn', 'npm'],
+          Browsers: ['Chrome', 'Edge', 'Firefox', 'Safari'],
+          npmPackages: ['@docusaurus/*/'],
```

Comment:
I dont think this will match against `@docusaurus/plugin-content-blog` or
`@docusaurus/core`
`@docusaurus/cli`

At least `@docusaurus/**` will match it.

That's why I mentioned it's important to try & test it out 😉

Reply:
@endiliey Thoughts?

Reply:
Since I had made a `symlink` in-order to test out the cli locally, it doesn't seem to recognise those packages and shows `not found`